docs/detections/api/rules/index-api-overview.asciidoc

```diff
@@ -3,14 +3,15 @@
 
 You use the index endpoint to create, get, and delete
 `.siem-signals-<Kibana-space>` system indices in a {kib} space.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment:
This one also looks a bit off:
![image](https://user-images.githubusercontent.com/616158/118679328-a7127b00-b7b2-11eb-8147-17bbb0aba8f4.png)